core/shark_turbine/importers/fx_importer.py

```diff
@@ -925,15 +928,21 @@ def _make_vtensor_literal_op(
         # buffer is via the indirection: Tensor -> list -> numpy array. This allows us to create a vtensor literal as
         # desired, but also limits which data types we can support in this function (see TORCH_DTYPE_TO_NPY_TYPE above)
         np_tensor = np.array(tensor.tolist()).astype(npy_dtype)
-        bytes_view = memoryview(np_tensor)
-        tensor_type = create_mlir_tensor_type(tensor)
-        shape_desc = "_".join([str(d) for d in tensor.shape])
-        blob_name = f"torch_tensor_{shape_desc}_{str(tensor.dtype)}"
-        elements_attr = DenseResourceElementsAttr.get_from_buffer(
-            bytes_view,
-            blob_name,
-            tensor_type,
-        )
+        # DenseResourceElementsAttr creation doesnt support rank 0 tensors, so we use DenseElementsAttr instead.
+        if np_tensor.size == 1:
+            elements_attr = DenseElementsAttr.get(
+                type=TORCH_DTYPE_TO_MLIR_TYPE[tensor.dtype](), array=np_tensor
```

Comment:
Can you also pass the shape= param? Keeps it from making dodgy local decisions and is marginally more robust.